src/JustEat.StatsD/StatsDMessageFormatter.cs

```diff
@@ -10,22 +10,19 @@ namespace JustEat.StatsD
 #endif
     public class StatsDMessageFormatter
     {
-        public const string SafeDefaultIsoCultureId = "en-US";
         private const double DefaultSampleRate = 1.0;
 
-        [ThreadStatic] private static Random _random;
+        [ThreadStatic]
+        private static Random _random;
 
         private readonly CultureInfo _cultureInfo;
         private readonly string _prefix;
 
-        public StatsDMessageFormatter() : this(new CultureInfo(SafeDefaultIsoCultureId), prefix: "") {}
-
-        public StatsDMessageFormatter(string prefix = "") : this(new CultureInfo(SafeDefaultIsoCultureId), prefix) {}
-
         public StatsDMessageFormatter(CultureInfo ci, string prefix = "")
```

Comment:
As nice as optional parameters are in code you own, they cause a nightmare upstream as adding/removing them are breaking changes, which can be made by accident easily. I suggest we use constructor overload here and pass ```string.Empty``` in from a new overload.